### DISCUSSION TITLE
Peer status is not displayed on iOS phones.

### DISCUSSION THREAD

**Discussion by vielang** [rustdesk/rustdesk#8638] [Q&A]
Peer status is not displayed on iOS phones. On Android phones, it displays normally. How can I fix it
![IMG_2322](https://github.com/rustdesk/rustdesk/assets/66666649/236a537a-6d56-41ae-a3a5-18db0cc3feaa)

**Comment by tschettervictor** [comment]
Duplicate of https://github.com/rustdesk/rustdesk/discussions/8658
